Tasks/Common/codecoverage-tools/jacoco/jacoco.maven.ccenabler.ts

```diff
@@ -81,11 +81,6 @@ export class JacocoMavenCodeCoverageEnabler extends cc.JacocoCodeCoverageEnabler
     protected addCodeCoverageNodes(buildJsonContent: any): Q.Promise<any> {
         let _this = this;
 
-        if (!buildJsonContent.project.build) {
```

Comment:
This block prevented the 'default' code from working in getBuildDataNode(), which caused the bug